src/midgard/tiles.cc

```diff
@@ -50,10 +50,10 @@ void bresenham_line(float x0,
 
 // a functor to generate closest first subdivisions of a set of tiles
 template <class coord_t> struct closest_first_generator_t {
-  coord_t seed;
   valhalla::midgard::Tiles<coord_t> tiles;
-  int32_t subcols, subrows;
+  coord_t seed;
   std::unordered_set<int32_t> queued;
+  int32_t subcols, subrows;
```

Comment:
my eyes cannot see the warning that was fixed here hahaha

Reply:
ha :) i guess i fixed `initialization order`by reordering members declaration order instead of reordering initialization order in the constructor(s)